src/boot/efi/boot.c

```diff
@@ -576,6 +578,8 @@ static void print_status(Config *config, char16_t *loaded_image_path) {
                         printf("        loader: %ls\n", entry->loader);
                 STRV_FOREACH(initrd, entry->initrd)
                         printf("        initrd: %ls\n", *initrd);
+                STRV_FOREACH(addon, entry->addons)
+                        printf("         addon: %ls\n", *addon);
```

Comment:
addon or add-on?

in the output here you spell it `addon` but in the type1 files you parse `add-on`. stick to one spelling. so far @bluca used `addon` mostly. I actually prefer `add-on`, but i guess it doesn't matter, hence maybe use `addon` everywhere then

Reply:
I prefer `add-on` too, but I don't want to spend bikeshedding time on that, feel free to decide for me :)

Reply:
The correct spelling is "Add-on" when in object form and "add on" when in verb form. Just like "(a) Log-in" vs "(to) log in".

Reply:
I prefer 'addon' so that it can always be used, if it was 'add-on' then every time it's used in code (like here) it would need to become 'add_on' give '-' is not a legal variable character

Reply:
Done